README.md

```diff
@@ -198,10 +199,10 @@ So far we've only been dealing with the standard output of a command, but what i
 ```wdl
 task hello {
   command {
-    echo 'hello ${name}!'
+    echo 'hello ${name}!' > test.out
   }
   output {
-    String response = read_string(stdout())
+    String response = read_string("test.out")
```

Comment:
This change raised an eyebrow... I'm pretty sure it should work the way it was originally.  I'll take a look.  Did you change this because it was broken for you or because you preferred this style over what was there?